qa/workunits/fs/misc/osd_full_mgr_hang.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -x
+
+set -e
+
```

Comment:
Please add an overview of what the test is doing in a comment. What behavior do you expect?